docs/antora.yml

```diff
@@ -10,6 +10,7 @@ asciidoc:
   attributes:
     # The full major.minor.patch version, which is used as a variable in the docs for things like download links
     full-version: '6.0.0-SNAPSHOT'
+    last-oss-version: '5.5.0'
```

Comment:
This probably needs a release pipeline change to keep it in sync as the versions are incremented (@nishaatr)

Reply:
can you pass the Slack link pls?

Reply:
May be this one https://hazelcastcommunity.slack.com/archives/C015Q2TUBKL/p1730371999914789

Reply:
FYI https://hazelcast.atlassian.net/browse/DI-321

Are we back-porting this to previous v/* branches?

Reply:
Should be on 5.4 and 5.5 I think. @oliverhowell has some magic labels to do it automatically ;)

Reply:
and 5.3?

Reply:
@oliverhowell whats "has some magic labels to do it automatically "? :)

Wondering if we need https://hazelcast.atlassian.net/browse/DI-321 after all

Reply:
> It's a user complain on Hazelcast Community Slack. And the user is right - we point to a version that does not exist.

Can you add the Slack link to the PR, please?

Reply:
I understand that currently it's wrong, but I think updating it _once_ without a plan on how it will be maintained in future (i.e. via an automation) is guaranteeing this will break in future.

Reply:
> Should be on 5.4 and 5.5 I think. @oliverhowell has some magic labels to do it automatically ;)

> @oliverhowell whats "has some magic labels to do it automatically "? :)

It's [this](https://github.com/hazelcast/hz-docs/blob/main/.github/workflows/backport.yml) - it allows _this_ change to backported, but don't update versions in the documentation, which is where [DI-321](https://hazelcast.atlassian.net/browse/DI-321) comes in.

Reply:
thanks @JackPGreen

Reply:
Guys - I agree that some automation may be necessary. IMHO best way is to merge this PR and create a DevInfra ticket to update release pipeline. Now we are just confusing users without any good reasons for them.

Reply:
> Guys - I agree that some automation may be necessary. IMHO best way is to merge this PR and create a DevInfra ticket to update release pipeline. Now we are just confusing users without any good reasons for them.

I reluctantly agree.

Reply:
I agree. Lets not block this. We have https://hazelcast.atlassian.net/browse/DI-321

But prefer to see response to https://github.com/hazelcast/hz-docs/pull/1355#issuecomment-2465223876 and if other places need updating then either we do it here or with another follow-up PR. Otherwise this PR feels incomplete